frontend/src/pages/storageClasses/StorageClassesPage.tsx

```diff
@@ -10,50 +10,78 @@ import {
   AlertActionCloseButton,
 } from '@patternfly/react-core';
 
-import { MetadataAnnotation, StorageClassConfig } from '~/k8sTypes';
+import { MetadataAnnotation } from '~/k8sTypes';
 import useStorageClasses from '~/concepts/k8s/useStorageClasses';
 import { ProjectObjectType, typedEmptyImage } from '~/concepts/design/utils';
 import ApplicationsPage from '~/pages/ApplicationsPage';
 import { updateStorageClassConfig } from '~/services/StorageClassService';
+import { useDeepCompareMemoize } from '~/utilities/useDeepCompareMemoize';
+import { ResponseStatus } from '~/types';
+import { StorageClassesTable } from './StorageClassesTable';
 import { getStorageClassConfig, isOpenshiftDefaultStorageClass } from './utils';
 
 const StorageClassesPage: React.FC = () => {
-  const [storageClasses, storageClassesLoaded, storageClassesError] = useStorageClasses();
   const [isAlertOpen, setIsAlertOpen] = React.useState(false);
-
+  const [isUpdating, setIsUpdating] = React.useState(false);
+  const [storageClasses, storageClassesLoaded, storageClassesError, refreshStorageClasses] =
+    useStorageClasses();
+  const storageClassesWithoutConfigs = useDeepCompareMemoize(
```

Comment:
Is there a reason to use `useDeepCompareMomoize` here? Is it possible to use `React.useMemo`? I think the `storageClasses` should be reference stable, right?

Reply:
It wasn't necessary, so just memoizing now as suggested.